utils/gradient.ts

```diff
@@ -9,11 +9,19 @@ export function djb2(str: string) {
 }
 
 export function generateGradient(username: string) {
-  const c1 = color({ h: djb2(username) % 360, s: 0.95, l: 0.5 });
-  const second = c1.triad()[1].toHexString();
+  const hue = djb2(username) % 360;
+
+  let hash2 = 7;
+  for (let i = 0; i < username.length; i++) {
+    hash2 = hash2 * 31 + username.charCodeAt(i);
+  }
+  const hue2 = Math.abs(hash2) % 360;
+
+  const c1 = color({ h: hue, s: 0.95, l: 0.5 });
+  const c2 = color({ h: hue2, s: 0.95, l: 0.5 });
```

Comment:
Can we keep the triad for c2?

Reply:
Hey @styfle,
It's done (https://github.com/vercel/avatar/pull/53/commits/1dc0c5f34f88b58ea74ca33ad109b989f17374c3), I've tried several approaches, but this seems to be the best one, tell me what you think.